tox.ini

```diff
@@ -9,9 +9,11 @@ skip_missing_interpreters = true
 
 [testenv]
 setenv = PYTHONPATH = {toxinidir}/src
-deps =
-    -r{toxinidir}/requirements.test.txt
-
+extras =
+    test
+allowlist_externals =
+    pytest
+    coverage
```

Comment:
I added these because I get the error below if aren't there when I run `tox -e py310`:

```
WARNING: test command found but not installed in testenv
  cmd: /home/user/github/rmartin16/briefcase/venv-3.10.7-rmartin16-briefcase/bin/coverage
  env: /home/user/github/rmartin16/briefcase/.tox/py310
Maybe you forgot to specify a dependency? See also the allowlist_externals envconfig setting.

DEPRECATION WARNING: this will be an error in tox 4 and above!
```